fix: sort channels correctly (#160)
NordicSemiconductor/pc-nrfconnect-dtm: Changelog.md

## 2.4.1 - Unreleased

### Fixed

> -   Sweep mode would in some casese not use the selected channels.

Reviewer: ```suggestion
-   Sweep mode in some cases wasn't using the selected channels.
```
